### PR TITLE
Mention alternative (deco) provisioning for tplink

### PR DESCRIPTION
**Pull request by apbarratt**
## Proposed change

Clarified that Tapo and Kasa devices that have been set up by other TP-Link apps (such as Deco), work correctly with this integration.

This came from my setting up of some new Tapo devices inside of the Deco app (which I use to manage my Deco wifi routers).  I previously had some devices set up using the Tapo app and a different TP-Link Cloud account (woo, relationship breakdowns in a digital world are complicated!) and had been putting off the migration as I wasn't sure if they'd work if I'd set them up in the Deco app instead of the Tapo one.  Now I've given it a go and see that it works I figured I'd let others who might be unsure know by clarifying in the docs that only mention to two apps used for the specific product lines in question :)

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].
  _I'm not entirely certain if "TP-Link Could Account" or "Deco app" are the preferred terms in the TP-Link's own marketing language, so these are best guess terms._

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Updated the `tplink` integration documentation to clarify the use of the Deco, Kasa, and Tapo apps for device provisioning.
	- Revised information on "Total consumption" reporting for Kasa and Tapo devices, including guidance on handling unavailable entities.
	- Modified the "Update" section to inform users about the removal of the entity due to stability issues and future replacements.

### CODE REVIEW

File: source/_integrations/tplink.markdown [hunk 1]
```diff
@@ -40,7 +40,7 @@ ha_integration_type: integration
 
 The `tplink` integration allows you to control your [TP-Link Kasa Smart Home Devices](https://www.tp-link.com/kasa-smart/) and [TP-Link Tapo Devices](https://www.tapo.com/) such as plugs, power strips, wall switches and bulbs.
 
-You need to provision your newly purchased device to connect to your network before it can be added via the integration. This can be done either by using [kasa command-line tool](https://python-kasa.readthedocs.io/en/latest/cli.html#provisioning) or by adding it to the official Kasa or Tapo app before trying to add them to Home Assistant.
+You need to provision your newly purchased device to connect to your network before it can be added via the integration. This can be done either by using [kasa command-line tool](https://python-kasa.readthedocs.io/en/latest/cli.html#provisioning) or by adding it to the official Kasa or Tapo app before trying to add them to Home Assistant. Some apps for TP-Link's other products, such as the Deco app, also allow you to add Kasa and Tapo devices within them, these devices use the same TP-Link Cloud Account for authorisation and so work with this integration also.
```
Comment (apbarratt) [inline source/_integrations/tplink.markdown]: Just realised, I've used International English spelling here for "authorisation".  Should I assume I should change this to the Oxford spelling, "authorization" to match with US English?  I'm not seeing a mention of language in the standards but I know US English seems to be the most common practice (frustratingly;).

Comment (apbarratt) [reply source/_integrations/tplink.markdown]: Ah, found it now, I was looking for "US English" but it says "American English", fixing now.

Comment (apbarratt) [reply source/_integrations/tplink.markdown]: Done.

**Review by rytilahti — APPROVED** [review]
Thanks @apbarratt, glad to hear alternative provisioning methods also work!

**Pull request merged**